### ISSUE TITLE
The styles for the card pattern (search variant) and BCL are not consistent

### ISSUE THREAD

**Issue by msnassar** [openeuropa/oe_bootstrap_theme#319]
It seems that the styles applied to the card pattern (search variant) is not aligned with the styles in the BCL.
### **Card pattern:**
```
<div class="row">
<div class="col-md-3 col-lg-2 rounded mw-listing-img">
<div class="col-md-9 col-lg-10">
</div>
```
**Font style: fs-4**
**_No space between the image/date block and the title_**
https://digit-oesa.acc.fpfis.tech.ec.europa.eu/en/node/47
![image](https://user-images.githubusercontent.com/15343499/226300890-4a54407e-f803-4c27-b6f2-7972f9957b71.png)

------------------------------------------------------------------------
### **BCL:**
```
<div class="row">
<div class="bcl-card-start-col mb-3 mb-lg-0 bcl-size-small">
<div class="col-12 col-md">
</div>
```
**Font style: fs-5**
![image](https://user-images.githubusercontent.com/15343499/226300619-dcab4d25-ac1f-4d8c-a5e5-fe3e32a3c351.png)

**_I think the card pattern should be updated to align with BCL_**

**Comment by msnassar** [comment]
Hello @tibi2303 and @brummbar 
After the update we still have no space between the image/date-block the the content
![image](https://github.com/openeuropa/oe_bootstrap_theme/assets/15343499/87930c70-431d-40b0-b8f3-bd1953ede0dd)

Here is our settings:

- [ ] Card image hidden on mobile
- [x] Card to use grid classes

**Comment by AaronGilMartinez** [comment]
Fixed in #399 and #426
